### PR TITLE
Removed unnecessary FQDNs from ApplicationRuleCollectionGroup as they are included by AzureKubernetesService FQDN Tag

### PR DESCRIPTION
**Pull request by magrande**
Background info:

https://docs.microsoft.com/en-us/azure/aks/limit-egress-traffic

### CODE REVIEW

**Review by ckittel — CHANGES_REQUESTED** [review]
Just a couple questions

File: networking/hub-regionA.json [hunk 1]
```diff
@@ -577,143 +532,6 @@
...
-                                        "terminateTLS": false,
-                                        "sourceAddresses": [],
-                                        "destinationAddresses": [],
-                                        "sourceIpGroups": [
-                                            "[resourceId('Microsoft.Network/ipGroups', variables('aksIpGroupName'))]"
-                                        ]
-                                    },
-                                    {
-                                        "ruleType": "ApplicationRule",
-                                        "name": "azure-monitor-addon",
```
[the remark this answers is not in the document]
Comment (magrande) [reply networking/hub-regionA.json:673]: @ckittel  The only tag not included in AKS tag is "eastus2.monitoring.azure.com", so I guess I should put it back, leaving only that one ?

[the remark this answers is not in the document]
Comment (ckittel) [reply networking/hub-regionA.json:673]: well, we can't have `eastus2` hard coded in here--that needs to be based on the provided location.  But my biggest concern is that you validated that the monitoring plugin was working without these?  According to the list of items included in the tag, I'm not seeing any of these addresses.

[the remark this answers is not in the document]
Comment (magrande) [reply networking/hub-regionA.json:673]: Done, we are playing sage and keeping the addons set of rules, and eastus2 is no longer hardcoded (using region parameter instead)

**Review by ckittel — APPROVED** [review]
🚀  Nice cleanup.

File: networking/hub-regionA.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (ckittel) [reply networking/hub-regionA.json:673]: Looks good.

File: networking/hub-regionA.json [hunk 2]
```diff
@@ -511,51 +511,6 @@
...
-                                        "terminateTLS": false,
-                                        "sourceAddresses": [],
-                                        "destinationAddresses": [],
-                                        "sourceIpGroups": [
-                                            "[resourceId('Microsoft.Network/ipGroups', variables('aksIpGroupName'))]"
-                                        ]
-                                    },
                                     {
                                         "ruleType": "ApplicationRule",
                                         "name": "management-plane",
```
[the remark this answers is not in the document]
Comment (ckittel) [reply networking/hub-regionA.json]: Thanks!

**Pull request merged**